public/app/features/plugins/sandbox/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { SandboxedPluginObject } from './types';
+
+export function isFunction(value: unknown): value is Function {
```

Comment:
Lodash version is not type-guarded and this version is. The lodash version works because it returns an any type. Though in this specific case it makes no difference. I'll change it to lodash